zipkin/src/main/scala/org/http4s/zipkin/models/ZipkinInfo.scala

```diff
@@ -0,0 +1,35 @@
+package org.http4s.zipkin.models
+
+import java.time.Instant
+
+import org.http4s.headers.{`X-B3-ParentSpanId`, `X-B3-SpanId`, `X-B3-TraceId`}
+
+final case class Endpoint(
+  ipv4: String,
```

Comment:
This isn't the first place we've needed a general ipv4 type.  We use the java.net one in a few places, but I'm always terrified of triggering a DNS lookup.  Something for us all to think about.